src/tests/ftest/soak/soak.py

```
@@ -0,0 +1,309 @@
+        self.server_group = self.params.get("server_group", '/server/',
+                                            'daos_server')
+        print("Servers {} group {} basepath {}".format(self.hostlist_servers,
+                                                       self.server_group,
+                                                       self.basepath))
+        ServerUtils.runServer(filename, self.server_group, self.basepath)
+
+        # setup the storage
+        self.create_pool()
+
+    def tearDown(self):
+        ServerUtils.stopServer(hosts=self.hostlist_servers)
+
+    def test_soak_1(self):
+        """
+        Test ID: DAOS-2192
+        Test Description: This test runs 2 DAOS API IOR jobs.
+        :avocado: tags=soak1
+        """
+
+        try:
+            # turn job parameters into slurm script
+            script1 = self.build_ior_script('job1')
+
+            # queue it up to run and register a callback to retrieve results
+            job_id1 = slurm_utils.run_slurm_script(script1)
+            slurm_utils.register_for_job_results(job_id1, self, maxwait=3600)
+
+            # queue up a second job
+            script2 = self.build_ior_script('job2')
+            job_id2 = slurm_utils.run_slurm_script(script2)
+            slurm_utils.register_for_job_results(job_id2, self, maxwait=3600)            
+
+            # wait for all the jobs to finish
+            while len(self.soak_results) < 2:
+                time.sleep(10)
+
+            for job, result in self.soak_results.iteritems():
+                if result != "COMPLETED":
+                    self.fail("Soak job: {} didn't complete as expected: {}".
+                              format(job, result))
+
+        except (DaosApiError, IorUtils.IorFailed) as error:
+            self.fail("<Soak Test 1 Failed>\n {}".format(error))
+        finally:
+            try:
+                os.remove(script1)
+            except StandardError:
```

Why StandardError? Is the idea to catch all Exceptions but not use the Exception keyword? I think something like OSError might fit better depending on the goal of these nested try blocks.